client/src/components/EmotionForm.jsx

```diff
@@ -136,16 +135,22 @@ const EmotionForm = ({ audioFeatures, savedTracks }) => {
           </div>
         </div>
       ) : null}
+      {/* conditionally render decision component */}
       {formData.selected_track ? (
         <div>
           <Decision handleDecision={handleDecision} />
         </div>
       ) : null}
+      {/* conditionally render EmoEntry component and submit button */}
```

Comment:
I like how you add nested comments in the jsx fragment itself! i haven't seen that before :)